introspection_msgs/msg/ROSMeta.msg

```diff
@@ -0,0 +1,2 @@
+string node_name 	# ROS node name
+uint8[] id 			# DDS ID
```

Comment:
Can you add some documentation to this message? A high level description would be helpful. And in particular why is the `id` field a vector? The short comment suggests that it's one ID.

And the name is not descriptive. Such that you only get a sense of the fact that it has metadata, but it clearly does not have all the metadata. It looks right now to be a name to id mapping.